cmd/parse.go

```diff
@@ -21,15 +27,19 @@ func getConfigStringValueFrom(flagValue, envValue, defaultValue string) string {
 	return defaultValue
 }
 
-func parseEnabledChecksValue() []string {
+// parseEnabledChecksValue returns the enabled checks requested by the user, or the defaultChecks
+// provided if the user has not provided any input.
+func parseEnabledChecksValue(defaultChecks []string) []string {
```

Comment:
I don't know if I am being too picky: I find it a bit confusing to pass around the default values as the arg. I believe we better define defaultChecks as a global var, and set it in the `RunE` function, and access this var here. Looking at the method name, I was expecting that the function is parsing the arg.

The other workaround is to pass both `userEnabledChecks` and `defaultChecks`.

Reply:
And I'm voting for removing parse.go altogether if possible.

Reply:
@samira-barouti We have two subcommands that need to parse enabled checks, but they have different default values. The alternative here was to write two functions, one for each, but that seemed inefficient.

Reply:
Shoudl be sorted. `parse.go` is removed, which, I think, resolves this thread 👍